opengate/managers.py

```diff
@@ -1668,17 +1670,74 @@ def _run_simulation_engine(self, start_new_process):
         with SimulationEngine(self) as se:
             se.new_process = start_new_process
             se.init_only = self.init_only
+            se.process_index = process_index
             output = se.run_engine()
         return output
 
-    def run(self, start_new_process=False):
+    def generate_run_timing_interval_map(self, number_of_processes):
+        if number_of_processes % len(self.run_timing_intervals) != 0:
```

Comment:
why ? I thought we just divide ALL time_interval by the number_of_processes

Reply:
Yes, but letting the user define the total number of processes rather than the process per run is more intuitive and will not require an API change if we implement a more advanced splitting scheme in the future. So I think it's better this way.